app/pages/lab-pages-editor/components/WorkflowSettingsPage/components/AssociatedSubjectSets.jsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react';
+
+const ARBITRARY_PAGE_SIZE = 250;  // If a project has more than this number of subject sets, then we need a better solution.
+
+export default function AssociatedSubjectSets({ project, workflow }) {
+  const [apiData, setApiData] = useState({
+    subjectSets: null,
+    status: 'ready'
+  });
+  const [linkedSubjectSets, setLinkedSubjectSets] = useState(workflow?.links?.subject_sets || []);
+
+  useEffect(() => {
+    async function fetchSubjectSets() {
+      try {
+        setApiData({
+          subjectSets: null,
+          status: 'fetching'
+        });
+
+        const ssets = await project.get('subject_sets', { sort: '+id', page_size: ARBITRARY_PAGE_SIZE });
+        if (!ssets) throw new Error('No subject sets');
+
+        setApiData({
+          subjectSets: ssets,
+          status: 'ready'
+        });
+
+      } catch (err) {
+        console.error('AssosicatedSubjectSets: ', err);
```

Comment:
Misspelling and would be useful to indicate its an error in the console.log(). I personally like using the following format:

`console.log("AssociatedSubjectSets() fetchSubjectSets() error", err);`

That way you have a guide in the console on how to find that console log (my future self really appreciates these guideposts). Only a suggestion though.

Reply:
Derp! I definitely need to fix that misspelling, thank you.

And thank you for the suggestion, I might also need to revise how I write our error messages. (At the moment I'm making the standards up as I go.)